### DISCUSSION TITLE
Tip for waiter

### DISCUSSION THREAD

**Discussion by ahorasoyfelipe** [Blair2004/NexoPOS#1033] [Ideas]
The possibility of working with a tip for a waiter, I thought that for now there would be the possibility of working it as if it were a tax, but with the possibility of the amount being editable

![Captura de Pantalla 2022-09-01 a las 01 57 59](https://user-images.githubusercontent.com/80654041/187842420-92401984-f521-4ac6-a713-b6d3abc9d840.png)

**Comment by Blair2004** [comment]
Hi, thank you for the suggestion.
Regarding tips for waiter, we're undecided, if that should be added to the commission module.
